gapic/src/main/com/google/gapid/Main.java

```diff
@@ -172,14 +172,16 @@ public void dispose() {
   private static final Flag<?>[] ALL_FLAGS = {
     Flags.help,
     Flags.version,
-    GapiPaths.gapidPath,
+    Follower.logFollowRequests,
```

Comment:
Please don't re-order the flag. They were ordered by importance/relevance to most users, as they are shown in this same order for --help.